kernel-build

```diff
@@ -1,44 +1,33 @@
-install deps:
-# dnf -y install gcc flex make bison openssl openssl-devel elfutils-libelf-devel ncurses-devel bc git tar
+Guide for kernel build.
 
-get the mainline kernel:
-# git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/
+install dependencies:
+# dnf -y install gcc flex make bison openssl openssl-devel elfutils-libelf-devel ncurses-devel bc git tar dwarves 
 
-copy over the config and validate it:
-# cp /boot/config-5.8.0-1.fc33.aarch64 .config
-# make olddefconfig
+get kernel from sensehat branch:
+# git clone https://github.com/underground-software/linux.git
 
-keep it small if you don't have space
-# make localmodconfig
+copy over config and validate it: 
+# cp /boot/config-5.11.12-200.fc33.aarch64 .config
```

Comment:
I'd add a note saying that this is an example from fedora and maybe say config-\<kernel-version\>.aarch64 instead of having a specific version